memento-cassandra.root/usr/share/clearwater/cassandra-schemas/memento.sh

```diff
@@ -1,9 +1,16 @@
 #! /bin/bash
 . /usr/share/clearwater/cassandra_schema_utils.sh
 
+cassandra_hostname="127.0.0.1"
+
+. /etc/clearwater/config
+
 quit_if_no_cassandra
 
-if [[ ! -e /var/lib/cassandra/data/memento ]];
+CQLSH="/usr/share/clearwater/bin/run-in-signaling-namespace cqlsh $cassandra_hostname"
+
+if [[ ! -e /var/lib/cassandra/data/memento ]] || \
+   [[ $cassandra_hostname != "127.0.0.1" ]];
```

Comment:
Why not do this if Cassandra is local? (If 127.0.0.1 is meant as some kind of "this is unconfigured" value, something like "UNCONFIGURED" seems better.)